tfx/components/statistics_gen/executor.py

```diff
@@ -106,27 +111,37 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
                 artifact_utils.get_single_uri(input_dict[SCHEMA_KEY])))
         stats_options.schema = schema
 
+    # Load and deserialize exclude splits from execution properties.
+    exclude_splits = json_utils.loads(
+        exec_properties.get(EXCLUDE_SPLITS_KEY)) or []
```

Comment:
json_utils.loads(exec_properties.get(EXCLUDE_SPLITS_KEY, 'null')) or [], same elsewhere

check my comments on the evaluator PR